kolibri/core/assets/src/navigation.vue

```diff
@@ -1,27 +1,26 @@
 <template>
-  <header role="banner">
-    <nav class="titlebar" role="navigation" aria-label="Main navigation">
-      <ul aria-label="User options submenu">
-        <li v-for="item in nav_items">
-          <a href="{{item.url}}">{{item.text}}</a>
-        </li>
-      </ul>
-    </nav>
-    </header>
+  <nav class="titlebar" role="navigation" aria-label="Main navigation">
+    <ul aria-label="User options submenu">
+      <li v-for="(url, label) in nav_items">
+          <a :style="{fontWeight: url == app ? 'bold' : ''}" :href=" '/' + url">
+            {{label}}
+          </a>
+      </li>
+    </ul>
+  </nav>
 </template>
 
 
 <script>
+  /* global _navItems window */
 
   module.exports = {
     data() {
       return {
         // items that go in the title bar
-        nav_items: window._nav ? window._nav.nav_items || [] : [],
-        title_bar: {
-          title: 'Kolibri',
-          home_link: '/',
-        },
+        nav_items: _navItems || [],
```

Comment:
If you change this to:
`nav_items: global._navItems || [],` it should solve your testing woes (note, it was failing on *all* versions of node that we test on).